updatePomVersions.sh

```diff
@@ -27,9 +27,6 @@ fi
 
 for i in samples/*
 do
-  (
-    cd $i
-    mvn versions:set -DnewVersion=$SDK_VERSION
-    rm pom.xml.versionsBackup
-  )
-done
\ No newline at end of file
+  sed  -i .versionsBackup "s/<akkaserverless-sdk.version>\(.*\)<\/akkaserverless-sdk.version>/<akkaserverless-sdk.version>$SDK_VERSION<\/akkaserverless-sdk.version>/" $i/pom.xml
+  rm $i/pom.xml.versionsBackup
```

Comment:
According to sed docs, one could omit the extension when using -i, but it didn't work for me. 

Also, the recommended usage is to create a backup file. We don't really need it, so I delete it as soon as we know that the file was written.